lib/jormungandr/test/integration/Test/Integration/Jormungandr/Scenario/API/Transactions.hs

```diff
@@ -244,101 +252,153 @@ spec = do
             }|]
         return (wSrc, payload)
 
-    fixtureExternalTx ctx toSend = do
-        -- initial source wallet
-        let initial = 50000 :: Natural
-        wSrc <- fixtureWalletWith ctx [initial]
-
-        -- source empty wallet which is going to take one 100 money inflow from
-        -- wSrc and from which we are going to send transaction externally to
-        -- another empty wallet, wEmpt
-        let password = "Secure Passphrase" :: Text
-        let createWallet = Json [json| {
-                "name": "1st Wallet",
-                "mnemonic_sentence": #{mnemonics15},
-                "passphrase": #{password}
-                } |]
-        r0 <- request @ApiWallet ctx ("POST", "v2/wallets") Default createWallet
-        verify r0
-            [ expectResponseCode @IO HTTP.status202
-            , expectFieldEqual walletName "1st Wallet"
-            , expectFieldEqual balanceAvailable 0
-            , expectFieldEqual balanceTotal 0
-            ]
-        let wDest = getFromResponse Prelude.id r0
-        let (Right seed) = fromMnemonic @'[15] @"seed" mnemonics15
-        let pwd = Passphrase $ BA.convert $ T.encodeUtf8 password
-        let rootXPrv = generateKeyFromSeed (seed, mempty) pwd
-        let st = mkSeqState @(Jormungandr 'Testnet)
-                (rootXPrv, pwd) defaultAddressPoolGap
-        let (addrD, st') = genChange pwd st
-        addrs <- listAddresses ctx wDest
-        let destination = (addrs !! 1) ^. #id
-        let amt = 100 :: Natural
-        let payload = Json [json|{
-                "payments": [{
-                    "address": #{destination},
-                    "amount": {
-                        "quantity": #{amt},
-                        "unit": "lovelace"
-                    }
-                }],
-                "passphrase": "Secure Passphrase"
-            }|]
-        let (feeMin, _feeMax) = ctx ^. feeEstimator $ TxDescription
-                { nInputs = 1
-                , nOutputs = 1
-                }
-        r1 <- request @(ApiTransaction t) ctx (postTxEp wSrc) Default payload
-        verify r1
+    postExternalTxTest ctx headers payload = do
+        let toSend = 1 :: Natural
+        (ExternalTxFixture wSrc wDest fee txWits) <-
+                fixtureExternalTx ctx toSend
+        let encodedSignedTx = encodeTx txWits MsgTypeTransaction
+        r <-
+            request @ApiTxId ctx postExternalTxEp headers (payload encodedSignedTx)
```

Comment:
This line is still too long!

https://github.com/input-output-hk/cardano-wallet/wiki/Coding-Standards#limit-line-length-to-80-characters

Reply:
oh, thanks for that. I have just installed 80-character line to my emacs config and see this. Should not have problems like that in the future